1.7/administration/installing/custom/advanced.md

```diff
@@ -46,6 +46,7 @@ The DC/OS installation creates these folders:
     cluster_name: '<cluster-name>'
     exhibitor_storage_backend: static
     ip_detect_filename: /genconf/ip-detect
+    master_discovery: static 
```

Comment:
I dunno if this master_discovery was required for 1.7 ... infact installs ran without any problem without master_discover up until 1.8.3 

1.8.4 testing was the first time I ran into a problem because my config.yaml did not have master_discovery